File: TYPESCRIPT/Cap07/c07ex07.ts
// c07ex07.ts

import * as rls from 'readline-sync';

function IGUAL(A: number, B: number): boolean {
  return A == B;
}

let X: number, Y: number;

X = rls.questionInt("Informe o 1o. valor: ");
Y = rls.questionInt("Informe o 2o. valor: ");

if (IGUAL(X, Y)) {
  console.log("Valores sao iguais");
} else {
  console.log("Valores sao diferentes");
}

console.log("");
process.stdout.write("Tecle <Enter> para encerrar... ");
rls.question();

File: LUA/Cap07/c07ex07.lua
-- c07ex07.lua

function igual(a, b)
  if (a == b) then
    return true
  else
    return false
  end
end

io.write("Informe o 1o. valor: ")
local x = tonumber(io.read())

io.write("Informe o 2o. valor: ")
local y = tonumber(io.read())

if (igual(x, y)) then
  print("Valores sao iguais")
else
  print("Valores sao diferentes")
end

io.write("\nPressione <Enter> para encerrar... ")
io.read()
